userdocs/contents/disks.rst

```diff
@@ -84,12 +85,29 @@ A swap partition with 1Gig can be added with
 
    wwctl node set n01 \
      --diskname /dev/vda \
-     --partname swap --partsize=1024 --partnumber 1 \
+     --partname swap --partsize=1024 --partnumber 2 \
      --fsname swap --fsformat swap --fspath swap
 
 which has the partition number ``1`` so that it will be added before the
 ``/scratch`` partition.
```

Comment:
This note here says that `swap` has partition number 1, but it has `--partnumber 2` in the example.